exoctk/exoctk_app/app_exoctk.py

```diff
@@ -510,10 +533,17 @@ def contam_visibility():
 
                 contam_script = contam_div = contam_js = contam_css = ''
 
-            return render_template('contam_visibility_results.html', form=form, vis_plot=vis_div,
-                                   vis_table=visib_table, vis_script=vis_script, vis_js=vis_js,
-                                   vis_css=vis_css, contam_plot=contam_div, contam_script=contam_script,
-                                   contam_js=contam_js, contam_css=contam_css)
+                return render_template('contam_visibility_results.html',
+                                       form=form, vis_plot=vis_div,
+                                       vis_table=visib_table,
+                                       vis_script=vis_script, vis_js=vis_js,
+                                       vis_css=vis_css, contam_plot=contam_div,
+                                       contam_script=contam_script,
+                                       contam_js=contam_js,
+                                       contam_css=contam_css,
+                                       tname=contamVars['tname'],
```

Comment:
Do you need to pass 'tname' and 'iname' to the template? You should be able to pull them from the form on the HTML page with `{{ form.targname.data }}` and `{{ form.inst.data }}`.